masonry_winit/src/event_loop_runner.rs

```diff
@@ -508,7 +513,23 @@ impl MasonryState<'_> {
             }
             WinitWindowEvent::RedrawRequested => {
                 let _span = info_span!("redraw");
-                self.render_root.handle_window_event(WindowEvent::AnimFrame);
+
+                let now = Instant::now();
+                // TODO: this calculation uses wall-clock time of the paint call, which
+                // potentially has jitter.
+                //
+                // See https://github.com/linebender/druid/issues/85 for discussion.
+                let last = self.last_anim.take();
+                let elapsed = last.map(|t| now.duration_since(t)).unwrap_or_default();
+
+                self.render_root
+                    .handle_window_event(WindowEvent::AnimFrame(elapsed));
+
+                // If this animation will continue, store the time.
+                // If a new animation starts, then it will have zero reported elapsed time.
+                let animation_continues = self.render_root.needs_anim();
+                self.last_anim = animation_continues.then_some(now);
+
```

Comment:
Would sort of rather see this live somewhere that it is shared between windowing integrations perhaps ... this isn't really specific to winit itself. Again, good reason to have these animation changes in a separate PR.